openfe/setup/protein_molecule.py

```diff
@@ -0,0 +1,59 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe
+from rdkit import Chem
+
+from openfe.utils.typing import RDKitMol, OEMol
+
+
+class ProteinMolecule:
+    """Wrapper around a Protein representation
+
+    This representation is immutable.  If you want to make any modifications,
+    do this in an appropriate toolkit then remake this class.
+    """
+    def __init__(self, rdkit: RDKitMol, name=""):
+        self._rdkit = rdkit
+        self._name = name
+
+    @property
+    def name(self):
+        return self._name
+
+    @classmethod
+    def from_pdbfile(cls, pdbfile: str, name=""):
+        m = Chem.MolFromPDBFile(pdbfile)
```

Comment:
We'll need to keep hydrogens if they are already around? I'd prefer we not have to deal with protonation tools at this stage.

```suggestion
        m = Chem.MolFromPDBFile(pdbfile, removeHs=False)
```